README.md

```diff
@@ -10,14 +10,14 @@
 
 <!-- prettier-ignore-start -->
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[all-contributors-badge]: https://img.shields.io/badge/all_contributors-15-orange.svg?style=flat-square
+[all-contributors-badge]: https://img.shields.io/badge/all_contributors-15-orange.svg
```

Comment:
Using the new [badge syntax](https://allcontributors.org/docs/en/bot/faq) allows it to finally be visible
```suggestion
<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
[![All Contributors](https://img.shields.io/badge/all_contributors-15-orange.svg?style=flat-square)](#contributors)
<!-- ALL-CONTRIBUTORS-BADGE:END -->
```

Reply:
I think we tried this approach originally, but couldn’t show our badges inline (without also running into lint errors). Here we specify the image URL sooner and use the all-contributors badge template instead so that we can keep the badge updating (and not manual) while also keeping the badges inline with our other badges.